test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -2988,29 +2911,20 @@ def test_cgroup_sismom_resize_fail(self):
         # Check that the sister mom failed to update the job
         self.moms_list[1].log_match(
             "Job;%s;sister node %s.* failed to update job"
-            % (jid, self.hosts_list[1]),
+            % (jid, self.hosts_list[0]),
             starttime=stime, interval=2, regexp=True)
         # Because of resize hook reject Mom failed to update the job.
         # Check that job got requeued.
         self.server.log_match("Job;%s;Job requeued" % (jid), starttime=stime)
 
-    @requirements(num_moms=2)
+    @requirements(num_moms=3)
     def test_cgroup_msmom_resize_fail(self):
         """
         Verify that exec_vnode values are trimmed
         when execjob_launch hook prunes job via release_nodes(),
         exec_job_resize failure in mom superior,
         tolerate_node_failures=job_start
         """
-        # Test requires 2 nodes
-        if len(self.moms) < 2:
-            self.skipTest('Test requires at least two Moms as input, '
-                          'use -p moms=<mom1:mom2>')
-        self.load_config(self.cfg7)
```

Comment:
I don't think you mean to remove also the "self.load_config(..)" call.

Reply:
Good catch. Fixed.